.github/workflows/check-PRs.yml

```diff
@@ -386,25 +386,6 @@ jobs:
           name: ping-pong-test-logs
           path: ./ping-pong-test.log
 
-      - name: Export/Import state
```

Comment:
Sorry for reviewing this again. These tests look important, why are you removing them?

Reply:
I wasnt able to make this work on GHA